tests/nonbonded/test_nonbonded_mol_energy.py

```diff
@@ -155,7 +152,7 @@ def test_nonbonded_mol_energy_random_moves(box_size, num_mols, moves, precision,
 
     mol_by_mol_pot = klass(N, group_idxs, beta, cutoff)
 
-    def u_ref(x, box, params):
+    def u_ref(x, box):
```

Comment:
@badisa - `params` was a stale param that was never used, so I removed it here and in other places down below.